### ISSUE TITLE
[gulliver-modules] I3IterativeFitter does not support Python objects as services (Trac #1936)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1928]
In contrast to I3SingleFitter, the iterative fitter cannot handle Python objects as services directly, instead you have to add them to the context first. This is old style and should be fixed.

Migrated from https://code.icecube.wisc.edu/ticket/1936
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:12:55", 
    "description": "In contrast to I3SingleFitter, the iterative fitter cannot handle Python objects as services directly, instead you have to add them to the context first. This is old style and should be fixed.", 
    "reporter": "kkrings", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067175380821", 
    "component": "combo reconstruction", 
    "summary": "[gulliver-modules] I3IterativeFitter does not support Python objects as services", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2017-01-20T15:03:17", 
    "milestone": "", 
    "owner": "kkrings", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170228T15:49:44: 
* kkrings commented:

> In changeset 153596:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="153596"
> I3IterativeFitter now accepts Python instances as services. I3RandomServices still have to be passed as strings because of the by default used GSL pseudo-random number generators. Closes #1928.
> ```


* kkrings changed status from "new" to "closed"
* kkrings changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:12:55: 
* olivas commented:

> Milestone February 2017 deleted


* olivas changed milestone from "February 2017" to ""
